todo/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { markDone, addTodo } from './actions';
+
+const style = { textDecoration: 'line-through' };
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userInput: ''
+    }
+  }
+
+  handleSubmit = event => {
```

Comment:
`handleChange` seems to match the functionality for this method better than `handleSubmit`.
```suggestion
  handleChange = event => {
```